docs/content/en/docs/getting-started/install-k8s.md

```diff
@@ -86,7 +86,7 @@ To install and deploy Tetragon, run the following commands:
 ```shell
 helm repo add cilium https://helm.cilium.io
 helm repo update
-helm install tetragon ${EXTRA_HELM_FLAGS} cilium/tetragon -n kube-system
+helm install tetragon ${EXTRA_HELM_FLAGS[@]} cilium/tetragon -n kube-system
```

Comment:
Unfortunately, this does not seem to work for bash which is what I would guess many people use.

How about:
```
EXTRA_HELM_FLAGS="--set tetragon.hostProcPath=/procHost" # flags for helm install
helm install tetragon $(echo ${EXTRA_HELM_FLAGS}) cilium/tetragon -n kube-system
```

Maybe there is something better?

Reply:
Removing the double quotes when setting the `EXTRA_HELM_FLAGS` env var does the trick for me for both shells. Not sure why I kept them in my patch.

- **zsh**

```bash
➜  tetragon git:(feature-i-doc-fixes) ps -p $$                                                              
    PID TTY          TIME CMD
 158412 pts/1    00:00:07 zsh

➜  tetragon git:(feature-i-doc-fixes) EXTRA_HELM_FLAGS=(--set tetragon.hostProcPath=/procHost --set tetragon.foo=bar)

➜  tetragon git:(feature-i-doc-fixes) helm install tetragon ${EXTRA_HELM_FLAGS[@]} cilium/tetragon -n kube-system
NAME: tetragon
LAST DEPLOYED: Fri Jan 12 00:36:04 2024
NAMESPACE: kube-system
STATUS: deployed
REVISION: 1
TEST SUITE: None

➜  tetragon git:(feature-i-doc-fixes) helm get values tetragon -n kube-system                                      
USER-SUPPLIED VALUES:
tetragon:
  foo: bar
  hostProcPath: /procHost
```
- **bash**

```bash
gozek@nexus:~/go/src/github.com/tetragon (feature-i-doc-fixes) $ ps -p $$
    PID TTY          TIME CMD
 191941 pts/1    00:00:00 bash

gozek@nexus:~/go/src/github.com/tetragon (feature-i-doc-fixes) $ EXTRA_HELM_FLAGS=(--set tetragon.hostProcPath=/procHost --set tetragon.foo=bar)

gozek@nexus:~/go/src/github.com/tetragon (feature-i-doc-fixes) $ helm install tetragon ${EXTRA_HELM_FLAGS[@]} cilium/tetragon -n kube-system
NAME: tetragon
LAST DEPLOYED: Fri Jan 12 00:38:20 2024
NAMESPACE: kube-system
STATUS: deployed
REVISION: 1
TEST SUITE: None

gozek@nexus:~/go/src/github.com/tetragon (feature-i-doc-fixes) $ helm get values -n kube-system tetragon
USER-SUPPLIED VALUES:
tetragon:
  foo: bar
  hostProcPath: /procHost
```

An alternative is to use `setopt shwordsplit` for `zsh` only which does not split words by default. Also see:

1. https://zsh.sourceforge.io/FAQ/zshfaq03.html#l18
2. https://stackoverflow.com/questions/6715388/variable-expansion-is-different-in-zsh-from-that-in-bash

Reply:
We should probably move this discussion to https://github.com/cilium/tetragon/pull/1965/commits/768b350b52701862f224769b2eb367f2d65c4c6e.

Reply:
Thanks!

> Of course I am OK with this approach if you prefer it.

No preference as long is it works for both shells.